lib/ansible/plugins/filter/core.py

```diff
@@ -57,7 +57,7 @@
 
 display = Display()
 
-UUID_NAMESPACE_ANSIBLE = uuid.UUID('361E6D51-FAEC-444A-9079-341386DA8E2E')
+UUID_NAMESPACE_ANSIBLE = '361E6D51-FAEC-444A-9079-341386DA8E2E'
```

Comment:
I would not change this to a string. Just just the namespace value before it is used and convert if necessary.

Reply:
I thought I’d change it to a string, because it’s now passed as a kwarg to to_uuid5, but some guidance is very welcome on how to proceed best. I am not sure how you suggested to proceed?